observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -32,12 +32,12 @@ import explore.model.Focused
 import gem.Observation
 import explore.components.ui.GPPStyles
 import explore.components.undo.UndoButtons
+import TargetObsQueries._
 
 final case class TargetObsList(
-  targets:        List[SiderealTarget],
-  observations:   View[List[ExploreObservation]],
-  focused:        ViewOpt[Either[SiderealTarget.Id, ExploreObservation.Id]],
-  onTargetSelect: SiderealTarget.Id => Callback
+  targets:      List[SiderealTarget],
+  observations: View[List[ExploreObservation]],
+  focused:      View[Option[Either[SiderealTarget.Id, ExploreObservation.Id]]]
```

Comment:
Why the Either? Is this to indicate we either clicked on a given target or one of the obs attached to the target?

Reply:
Exactly. And the `Option` is for the case where we are in this page but no target or obs has been clicked yet.